backend/app/views/backoffice/projects/_filters.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render "backoffice/shared/filters", q: q do |f| %>
+  <%= render partial: "backoffice/shared/full_text_filter", locals: { f: f } %>
+  <div class="w-30 inline-block mx-3">
+    <%= f.input :status_eq, as: :select, required: false, label: false, include_blank: I18n.t("backoffice.common.status"),
+                collection: Project.statuses.map { |k, v| [I18n.t("activerecord.attributes.project.statuses.#{k}"), v] } %>
+  </div>
+  <div class="w-30 inline-block">
+    <%= f.input :category_eq, as: :select, required: false, label: false, include_blank: I18n.t("backoffice.common.category"),
+                collection: Category.all.sort_by(&:name).map { |c| [c.to_s, c.slug] } %>
```

Comment:
Is the `map` even necessary? I think at least for simple form collection it is not for EnumModels.